packages/backend-data/src/types.ts

```diff
@@ -139,6 +139,12 @@ export type DataProps = {
    * Functions invokable by the API. The specific input type of the function is subject to change or removal.
    */
   functions?: Record<string, ConstructFactory<AmplifyFunction>>;
+
+  /**
+   * ExperimentalStackMapping override the assigned nested stack on a per-resource basis. Only applies to resolvers, and takes the form
+   * { <logicalId>: <stackName> }
+   */
+  experimentalStackMapping?: Record<string, string>;
```

Comment:
We have not yet finalized a method to expose experimental APIs so please hold with this change.